release_docs/INSTALL_CMake.txt

```diff
@@ -837,6 +867,14 @@ if (WINDOWS)
 else ()
     H5_DEFAULT_PLUGINDIR    "/usr/local/hdf5/lib/plugin"
 endif ()
+if (BUILD_SZIP_WITH_FETCHCONTENT)
+    LIBAEC_TGZ_ORIGPATH       "Use LIBAEC from original location"        "https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.0.6"
```

Comment:
We may want to mirror this in GitHub. I'm under the impression that loading anything outside of GitHub can be very slow.

Reply:
@derobins @byrnHDF Note that one of the original authors of libaec has an active Github mirror here:
https://github.com/MathisRosenhauer/libaec